SELF_ASSESSMENT.md

```diff
@@ -164,15 +171,27 @@ for changes to the project.
 the project, such as deployment configurations, settings, etc.  These should also be
 included in threat modeling.--->
 
```

Comment:
This section is for "describe anything built into your project that is designed to improve the security for users". Does the component of the system needs to be described here? or something else?

"Critical elements are non-configurable design decisions intended to increase the security of the project."
How is OSCAL improving the security of the project.

"Security Relevant.  A listing of security relevant components of the project with
  brief description.  These are considered important to enhance the overall security of
the project, such as deployment configurations, settings, etc.  These should also be
included in threat modeling."
Do we need to describe any deployment configuration and settings that can be used to change the security settings?

Reply:
From the description of the critical and security relevant, it seems we need to describe what all things we have in our project that improves its security. Hence, I feel OSCAL sdk etc are project components and not specifically for security of the project itself.

But security aspects related to plugin architecture, GIT and CI/CD workflows and configurations, etc are what makes the project secure.